index.mako

```diff
@@ -1501,27 +1501,27 @@ you use pre-commit!
 
 - Markdown:
 
-  ```md
+  ```md#copyable
   [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
   ```
 
 - HTML:
 
-  ```html
+  ```html#copyable
   <a href="https://github.com/pre-commit/pre-commit"><img src="https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white" alt="pre-commit" style="max-width:100%;"></a>
   ```
 
 - reStructuredText:
 
-  ```rst
+  ```rst#copyable
   .. image:: https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white
      :target: https://github.com/pre-commit/pre-commit
      :alt: pre-commit
   ```
 
 - AsciiDoc:
 
-  ```
+  ``` #copyable
```

Comment:
I think this one needs the space removed iirc?

Reply:
it's optional, but I removed it